src/ralph/virtual/admin.py

```diff
@@ -514,7 +514,7 @@ class CloudProviderAdmin(RalphAdmin):
 @register(CloudImage)
 class CloudImageAdmin(RalphAdmin):
     list_display = ['name', 'image_id']
-    list_filter = ['name', 'image_id']
+    list_filter = ['image_id']
```

Comment:
After giving it a thought I think it's actually better to leave the name here and get rid of the uuid -- I can imagine someone recalling a part of the image name and trying to filter by that, however, remembering an image uuid is unlikely.

Reply:
Another point is that it would be useful to enable API filtering by both name and uuid fields. API filters are defined on view sets and by default are copied from admin. So in this case it's necessary to add a ```list_filter``` property to ```CloudImageViewSet```. Here you can find an example: https://github.com/allegro/ralph/blob/ng/src/ralph/virtual/api.py#L181

Reply:
Done.